prevent install of a bunch of useless stuff that is already linked in
--
diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9d41d0c..d42ab53 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -56,20 +56,20 @@ set(BUILD_SHARED_LIBS OFF)
 # Disable building tests inside QHotKey library
 set(CMAKE_POLICY_DEFAULT_CMP0077 NEW)
 set(QHOTKEY_EXAMPLES OFF)
-add_subdirectory(libraries/qhotkey)
+add_subdirectory(libraries/qhotkey EXCLUDE_FROM_ALL)
 
 # FakeVim library
 set(CREATE_STATIC_LIBRARY ON)
 if(MSVC)
     add_compile_definitions(FAKEVIM_STATIC_DEFINE QTCREATOR_UTILS_STATIC_LIB)
 endif()
-add_subdirectory(libraries/fakevim)
-add_subdirectory(libraries/diff_match_patch)
+add_subdirectory(libraries/fakevim EXCLUDE_FROM_ALL)
+add_subdirectory(libraries/diff_match_patch EXCLUDE_FROM_ALL)
 
 option(USE_QLITE_HTML "Build using QLiteHtml for preview" OFF)
 if (USE_QLITEHTML)
     add_definitions(-DUSE_QLITEHTML=1)
-    add_subdirectory(libraries/qlitehtml)
+    add_subdirectory(libraries/qlitehtml EXCLUDE_FROM_ALL)
 endif()
 
 
